R/cnd-abort.R

```diff
@@ -53,7 +54,14 @@
 #'
 #'   Can also be `NULL` or a [defused function call][topic-defuse] to
 #'   respectively not display any call or hard-code a code to display.
-#' @param body,footer Additional bullets.
+#' @param parent Supply `parent` when you rethrow an error from a
+#'   condition handler (e.g. with [try_fetch()]).
+#'
+#'   If `parent` is a condition object, a _chained error_ is
+#'   created. If `parent` is `NA`, it indicates an unchained rethrow.
```

Comment:
It might be useful to add additional context about when these two cases are useful. Something like:

If `parent` is a condition object, a _chained error_ is created, which is useful when you want to enhance an error with more details, while still retaining the original information.

If `parent` is `NA`, it indicates an unchained rethrow, which is useful when you want to take ownership over an error and rethrow it with a custom message that better fits the surrounding context.

Reply:
Good point and great suggestions! This param doc is now much better: https://github.com/r-lib/rlang/commit/5196cd3055d4f9d7c6c82747bc670074dbde50eb